test/e2e/framework/node/wait.go

```diff
@@ -84,7 +83,7 @@ func WaitForTotalHealthy(c clientset.Interface, timeout time.Duration) error {
 		}
 		missingPodsPerNode = make(map[string][]string)
 		for _, node := range nodes.Items {
-			if !system.DeprecatedMightBeMasterNode(node.Name) {
+			if isNodeSchedulableWithoutTainted(&node) {
```

Comment:
nit: I think maybe `isNodeSchedulableWithoutTaints` reads slightly better.

Reply:
@BenTheElder Thanks, that seems better for me. OK, I will update this.